sdk/documentintelligence/ai-document-intelligence-rest/README.md

```diff
@@ -267,6 +313,86 @@ console.log(response);
 //  }
 ```
 
+## Get analyze PDF
```

Comment:
I am not sure if I understand correctly but the title doesn't make sense. Should it be "Get analyed PDF" or just "Analyze PDF"?
```suggestion
## Analyze PDF
```

Reply:
This is to get the output PDF after an analysis.

Reply:
I'll call it `Get the generated PDF output from document analysis.`